benchmarks/DeepLearning/Ops/Conv2DNchwFchwOp/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+set(BUDDY_CONV_OPT_ATTR avx512f)
+set(LLVM_MLIR_BINARY_DIR ${BUDDY_CONV_OPT_BUILD_DIR}/../llvm/build/bin)
```

Comment:
We changed the name of the buddy-mlir opt tool.

${BUDDY_CONV_OPT_BUILD_DIR} -> ${BUDDY_OPT_BUILD_DIR}

Reply:
done